src/integrationTest/java/uk/gov/hmcts/reform/judicialapi/util/KeyGenUtil.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.judicialapi.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.jwk.RSAKey;
+import com.nimbusds.jose.jwk.gen.RSAKeyGenerator;
+import net.minidev.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class KeyGenUtil {
+
```

Comment:
why this needed? not getting

Reply:
It's a util class used by `JwtTokenUtil` to get RSAKey.